draft-irtf-cfrg-vdaf.md

```diff
@@ -2608,7 +2608,7 @@ def helper_proofs_share(Prio3, agg_id, k_share):
       Prio3.Flp.Field,
       k_share,
       Prio3.domain_separation_tag(USAGE_PROOF_SHARE),
-      byte(agg_id),
+      byte(Prio3.PROOFS) + byte(agg_id),
```

Comment:
Note to myself. `byte` actually returns `bytes`, which is a sequence. So `+` is concatenation operation.